.github/workflows/storybook_deploy_main.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy Main Storybook
+
+on:
+  schedule:
+    - cron: '0 7 * * *'
+
+jobs:
+  deploy_main_storybook:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        clean: false
```

Comment:
What's the benefit of this?

Reply:
This was leftover from previous iterations, it's probably not needed anymore.  Will remove